source/run-command.txt

```diff
@@ -0,0 +1,188 @@
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+}
+to run a database command. You can use database commands to perform a
+variety of administrative and diagnostic tasks, such as fetching server
+statistics, initializing a replica set, or running an aggregation pipeline.
+
+.. important:: Prefer Library Methods to Database Commands
+
+   The library provides wrapper methods for many database commands.
+   We recommend using these methods instead of executing database
+   commands when possible.
+   
+   To perform administrative tasks, use the :mongosh:`MongoDB Shell </>`
+   instead of the {+php-library+}. The shell provides helper methods
+   that might not be available in the library.
+   
+   If there are no available helpers in the library or the shell, you
+   can use the ``db.runCommand()`` shell method.
+
+.. _php-execute-command:
+
+Execute a Command
+-----------------
+
+To run a database command, you must specify the command and any relevant
+parameters in a command document, then pass the command document to the
+``MongoDB\Database::command()`` method. Many database commands return
+multiple result documents, so the ``command()`` method returns a
+``Cursor`` object that you can iterate through.
+
+The following code shows how you can use the ``command()``
+method on a :phpclass:`MongoDB\Database` instance to run the ``hello``
+command, which returns information about the current member's role in
+the replica set:
+
+.. literalinclude:: /includes/write/run-command.php
+   :language: php
+   :dedent:
+   :start-after: start-hello
+   :end-before: end-hello
+
+To find a link to a full list of database commands and corresponding
+parameters, see the :ref:`Additional Information section
+<php-addtl-info-runcommand>`.
+
+.. note:: Read Preference
+
+   The ``command()`` method does not obey the read preference you might
+   have set on your ``Database`` instance elsewhere in your code. By
+   default, ``command()`` uses the ``primary`` read preference.
+   
+   You can set a read preference for command execution by setting one
+   in the options parameter, as shown in the following code:
+   
+   .. literalinclude:: /includes/write/run-command.php
+      :language: php
+      :dedent:
+      :start-after: start-readpref
+      :end-before: end-readpref
+
+   For more information on read preference options, see :manual:`Read
+   Preference </core/read-preference/>` in the {+mdb-server+} manual.
+
+.. _php-command-response:
+
+Response
+--------
+
+The ``command()`` method returns a ``Cursor`` object that contains
+the response from the database for the given command. Each database
+command performs a different function, so the response
+content can vary. Some command responses contain multiple result
+documents. In these situations, the library converts the cursor
+envelope in the raw command response, which includes the cursor ID and
+the first batch of results, into an iterable cursor.
```

Comment:
Consider adding something like the following to cover the general case for commands that _don't_ return cursors:

> For commands that return a single result document, that result will be made available as the first and only document in the cursor.

Since most cursor-returning commands have helpers, I'd consider this case to be more common.